fern/ir-types/definition/services/http.yml

```diff
@@ -9,7 +9,10 @@ types:
     extends: commons.WithDocs
     properties:
       name: serviceCommons.DeclaredServiceName
-      basePath: optional<string>
+      basePath:
+        type: optional<string>
+        docs: This is deprecated.
+      basePathV2: optional<HttpPath>
```

Comment:
I think if the service is gonna have the HttpPath in the IR, should the path params go here too?

Reply:
down to do that